lib/network.ts

```diff
@@ -0,0 +1,25 @@
+import { networkInterfaces } from 'os'
+
+export function getLocalIpAddress(): string {
```

Comment:
Can you explain me what's the idea behind this logic? I thought that using current IP/hostname from the browser URL is just enough for self-hosting at home, no?